.github/workflows/molecule-test.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: "Molecule Test"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # access to check out code and install dependencies
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12.3'
```

Comment:
Can we use 3.12?

Reply:
Test fails without the patch number.

`Run actions/setup-python@v5
Installed versions
  Version 3.12 was not found in the local cache
  Error: The version '3.12' with architecture 'x64' was not found for this operating system.
  The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json`